pkg/kmm/kubelet.go

```diff
@@ -17,15 +17,26 @@ import (
 // CreateAndStartKubelet will call the CreateAndStartKubelet method with the correct configuration
 func (k *Kmm) CreateAndStartKubelet(master bool) error {
 
+	nodeLables := ""
+	if k.NodeLabels != nil {
+		for k, v := range k.NodeLabels {
+			if len(nodeLables) > 0 {
+				nodeLables = nodeLables + ","
+			}
+			nodeLables = nodeLables + k + "=" + v
```

Comment:
Could do like this: https://github.com/UKHomeOffice/keto/blob/master/pkg/keto/resource_printer.go#L96